classes/template/PKPTemplateManager.inc.php

```diff
@@ -1272,6 +1273,34 @@ function smartyAssign($value, $varName, $passThru = false) {
 	}
 
 	/**
+	 * Override the built-in smarty escape modifier to
+	 * add the jqselector escaping method.
+	 */
+	function smartyEscape($string, $esc_type = 'html', $char_set = 'ISO-8859-1') {
+		$pattern = "/(:|\.|\[|\]|,|=|@)/";
+		$replacement = "\\\\\\\\$1";
```

Comment:
I wonder whether this many backslashes are required because you're already escaping in the ID markup. That seems like too many.